[2021-02-23 14:57:42,820] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_branch_dop_operator_v3.condition 2021-02-21T00:01:00+00:00 [queued]>
[2021-02-23 14:57:44,584] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_branch_dop_operator_v3.condition 2021-02-21T00:01:00+00:00 [queued]>
[2021-02-23 14:57:44,626] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-23 14:57:44,644] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-02-23 14:57:44,698] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-23 14:57:45,696] {taskinstance.py:1063} INFO - Executing <Task(BranchPythonOperator): condition> on 2021-02-21T00:01:00+00:00
[2021-02-23 14:57:45,820] {standard_task_runner.py:52} INFO - Started process 139 to run task
[2021-02-23 14:57:46,136] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_branch_dop_operator_v3', 'condition', '2021-02-21T00:01:00+00:00', '--job-id', '40', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_branch_python_dop_operator_3.py', '--cfg-path', '/tmp/tmpndb2nd8d', '--error-file', '/tmp/tmpm974r1h1']
[2021-02-23 14:57:46,156] {standard_task_runner.py:77} INFO - Job 40: Subtask condition
[2021-02-23 14:57:47,785] {logging_mixin.py:104} INFO - Running <TaskInstance: example_branch_dop_operator_v3.condition 2021-02-21T00:01:00+00:00 [running]> on host 0513e3a4eab6
[2021-02-23 14:57:48,586] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_branch_dop_operator_v3
AIRFLOW_CTX_TASK_ID=condition
AIRFLOW_CTX_EXECUTION_DATE=2021-02-21T00:01:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-21T00:01:00+00:00
[2021-02-23 14:57:48,597] {logging_mixin.py:104} INFO - ------------- exec dttm = 2021-02-21T00:01:00+00:00 and minute = 1
[2021-02-23 14:57:48,615] {python.py:118} INFO - Done. Returned value was: dummy_task_2
[2021-02-23 14:57:48,651] {skipmixin.py:124} INFO - Following branch dummy_task_2
[2021-02-23 14:57:48,994] {skipmixin.py:155} INFO - Skipping tasks ['dummy_task_1']
[2021-02-23 14:57:50,054] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_branch_dop_operator_v3, task_id=condition, execution_date=20210221T000100, start_date=20210223T145743, end_date=20210223T145750
[2021-02-23 14:57:52,147] {local_task_job.py:188} WARNING - State of this instance has been externally set to success. Terminating instance.
[2021-02-23 14:57:52,341] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 139
[2021-02-23 14:57:52,469] {taskinstance.py:1239} ERROR - Received SIGTERM. Terminating subprocesses.
[2021-02-23 14:57:52,948] {process_utils.py:66} INFO - Process psutil.Process(pid=139, status='terminated', exitcode=1, started='14:57:44') (139) terminated with exit code 1
